catalog/forms.py

```diff
@@ -0,0 +1,64 @@
+from django import forms
+from django.contrib.auth import get_user_model
+
+from catalog.models import Cook, Dish
+
+
+class CookExperienceUpdateForm(forms.ModelForm):
+    class Meta:
+        model = Cook
+        fields = ["years_of_experience"]
+
+    def clean_years_of_experience(self):
+        return validate_years_of_experience(
+            self.cleaned_data["years_of_experience"])
+
+
+def validate_years_of_experience(value):
+    if value < 0:
+        raise forms.ValidationError("Years cannot be negative")
+    return value
+
+
+class CookSearchForm(forms.Form):
+    username = forms.CharField(
+        max_length=255,
+        required=False,
+        label="",
+        widget=forms.TextInput(attrs={"placeholder": "Search by username"}),
+    )
+
+
+class DishForm(forms.ModelForm):
+    cooks = forms.ModelMultipleChoiceField(
+        queryset=get_user_model().objects.all(),
```

Comment:
Jeśli są to instancje klasy Cook, to możemy je wyquerować w sposób jawny. Jest to bardziej czytelne
```suggestion
        queryset=Cook.objects.all(),
```